### ISSUE TITLE
[Components and pattern standards] Design components and patterns in use are outdated.  (04.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#61937]
#### VFS team name
10-10 Health Apps

#### VFS product name
10-10EZ

#### VFS feature name
Household Information Optimization

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components and patterns in use are outdated. 

### Issue Details
You are using the new [white card](https://design.va.gov/components/card#default---white-with-border) with border on your list and loop page. We have recently added this new style to the design system and it is slightly different from your UI. Also included in the new card style are tertiary buttons which are slightly different as well.

### Link, screenshot or steps to recreate
[Card image](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/47766-Allison-04-08-01.png)

### VA.gov Experience Standard
[Category Number 04, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Update your card UI to match the design system (border color color-gray-medium and no radius). We have also updated the tertiary button guidelines, make the "Remove" text all uppercase. See [FSR card example](https://design.va.gov/components/card#financial-status-report).

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by humancompanion-usds** [comment]
You can see the styling for the Button - Icon for Remove in this [Sketch file](https://sketch.com/s/be292806-8dbf-4201-bfa7-0edbba103347). Allison mentioned using uppercase and I'll add that it would be good to also check the hover, focus, and active states. This component should be going into the Design System via the experimental process very soon. Thanks!

**Comment by longmd** [comment]
@allison0034 , this has been updated and is ready for review and final sign-off.
